runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -64,6 +65,29 @@ static onert::ir::Layout convertLayout(NNFW_LAYOUT layout)
   return onert::ir::Layout::UNKNOWN;
 }
 
+NNFW_STATUS getTensorIndexImpl(onert::ir::IOIndex io_index, const char *tensorname, uint32_t *index)
+{
+  if (!tensorname || !index)
+    return NNFW_STATUS_UNEXPECTED_NULL;
+
+  if (!null_terminating(tensorname, MAX_TENSOR_NAME_LENGTH))
```

Comment:
(optional) It would be nice to move this check before finding `IOIndex` by `tensorname`.

Reply:
Nice catch!